<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DashboardWindow</class>
 <widget class="QMainWindow" name="DashboardWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Izvor - Dashboard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">

    <!-- Red 1 -->
    <item>
     <layout class="QHBoxLayout">
      <item><widget class="QPushButton" name="btnHome"><property name="text"><string>Home</string></property></widget></item>
      <item><widget class="QPushButton" name="btnStatistics"><property name="text"><string>Statistics</string></property></widget></item>
      <item><widget class="QPushButton" name="btnClosedPositions"><property name="text"><string>Closed Positions</string></property></widget></item>
      <item><widget class="QPushButton" name="btnExitAll"><property name="text"><string>Exit All</string></property></widget></item>
     </layout>
    </item>

    <!-- Red 2 -->
    <item>
     <layout class="QHBoxLayout">
      <item><widget class="QPushButton" name="btnAIStatistics"><property name="text"><string>AI Statistics</string></property></widget></item>
      <item><widget class="QPushButton" name="btnAIPrediction"><property name="text"><string>AI Prediction</string></property></widget></item>
      <item><widget class="QPushButton" name="btnLivePositions"><property name="text"><string>Live Positions</string></property></widget></item>
      <item><widget class="QPushButton" name="btnSettings"><property name="text"><string>Settings</string></property></widget></item>
     </layout>
    </item>

    <!-- Trening, Refresh, Combo Mode -->
    <item>
     <layout class="QHBoxLayout">
      <item><widget class="QPushButton" name="btnAITrain"><property name="text"><string>Pokreni AI Trening</string></property></widget></item>
      <item><widget class="QPushButton" name="btnRefresh"><property name="text"><string>Refresh</string></property></widget></item>
      <item><widget class="QComboBox" name="comboMode">
        <property name="editable"><bool>false</bool></property>
        <item><property name="text"><string>scalping</string></property></item>
        <item><property name="text"><string>daily</string></property></item>
      </widget></item>
      <item><widget class="QLabel" name="lblCurrentMode">
         <property name="text"><string>Trenutni mod: Scalping</string></property>
         <property name="alignment"><set>Qt::AlignCenter</set></property>
      </widget></item>
     </layout>
    </item>

    <!-- AI Prediction -->
    <item>
     <widget class="QLabel" name="lblAIPrediction">
      <property name="text"><string>AI Prediction: -</string></property>
      <property name="alignment"><set>Qt::AlignCenter</set></property>
     </widget>
    </item>

    <!-- Buy/Sell label (moÅ¾e da se izbaci ili sakrije) -->
    <item>
     <layout class="QHBoxLayout">
      <item><widget class="QLabel" name="labelBuy"><property name="text"><string>Buy</string></property><property name="alignment"><set>Qt::AlignCenter</set></property></widget></item>
      <item><widget class="QLabel" name="labelSell"><property name="text"><string>Sell</string></property><property name="alignment"><set>Qt::AlignCenter</set></property></widget></item>
     </layout>
    </item>

    <!-- START/STOP dugme -->
    <item>
     <widget class="QPushButton" name="btnStart"><property name="text"><string>START/STOP</string></property></widget>
    </item>

    <!-- Statusi + Checkboxes -->
    <item>
     <layout class="QHBoxLayout">
      <item>
       <layout class="QVBoxLayout">
        <item><widget class="QLabel" name="lblNewsSentiment"><property name="text"><string>News Sentiment</string></property></widget></item>
        <item><widget class="QLabel" name="lblTradingStatus"><property name="text"><string>Trading Status</string></property></widget></item>
        <item><widget class="QLabel" name="lblSignal"><property name="text"><string>Signal</string></property></widget></item>
        <item><widget class="QLabel" name="lblNews"><property name="text"><string>News</string></property></widget></item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout">
        <item><widget class="QCheckBox" name="checkPauseOnNews"><property name="text"><string>Pause On News</string></property></widget></item>
        <item><widget class="QCheckBox" name="checkShowOnlyMySymbol"><property name="text"><string>Show Only My Symbol</string></property></widget></item>
       </layout>
      </item>
     </layout>
    </item>

    <!-- Progress bar -->
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="minimum"><number>0</number></property>
      <property name="maximum"><number>100</number></property>
      <property name="value"><number>0</number></property>
      <property name="textVisible"><bool>true</bool></property>
      <property name="format"><string>Priprema u toku...</string></property>
     </widget>
    </item>

    <!-- Tabela na dnu -->
    <item>
     <widget class="QTableView" name="tableStats"/>
    </item>

   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
